index.js

```diff
@@ -1,10 +1,19 @@
 'use strict';
 
-var frames = process.platform === 'win32' ?
-	['-', '\\', '|', '/'] :
-	['⠋', '⠙', '⠹', '⠸', '⠼', '⠴', '⠦', '⠧', '⠇', '⠏'];
+var styles = {
+  win32: ['-', '\\', '|', '/'],
+  spinner: ['⠋', '⠙', '⠹', '⠸', '⠼', '⠴', '⠦', '⠧', '⠇', '⠏'],
+  bounce: ['⠁', '⠂', '⠄', '⠂']
+};
 
-module.exports = function () {
+module.exports = function (style) {
```

Comment:
I rather have an options object here in case we add additional options in the future.